src/Microsoft.Health.Fhir.SqlServer/Features/Schema/SchemaVersionConstants.cs

```diff
@@ -18,6 +18,7 @@ public static class SchemaVersionConstants
         public const int SupportsResourceChangeCaptureSchemaVersion = (int)SchemaVersion.V14;
         public const int BulkReindexReturnsFailuresVersion = (int)SchemaVersion.V16;
         public const int AddMinMaxForDateAndStringSearchParamVersion = (int)SchemaVersion.V18;
-        public const int SupportsPartitionedResourceChangeDataVersion = (int)SchemaVersion.V21;
+        public const int SupportsPartitionedResourceChangeDataVersion = (int)SchemaVersion.V20;
+        public const int ParameterizedRemovePartitionFromResourceChangesVersion = (int)SchemaVersion.V21;
```

Comment:
Am I missing something? I don't see this new constant being used anywhere

Reply:
Yes, it is not used on the OSS side. It is going to be used on Health PaaS.

Reply:
I am wondering if we can define this constant in health-paas instead as someone might still get confused that its not being used any where?

Reply:
I think we should keep constants in one place since it is related to the DB schema version. In the future, we might release Event Agent for the OSS.

Reply:
I would prefer to keep the constant here for future event support in OSS, just to keep everything in one place, even though it would not be used in OSS at this time.

Reply:
Adding a comment looks odd and might be confusing for OSS-only users. If possible, I would like to leave it as it is since it makes sense to each schema version has a corresponding descriptive constant.

Reply:
Adding a comment looks odd and might be confusing for the OSS-only users. I think it is safe to leave it as it is since it makes sense to each schema version has a corresponding descriptive constant.

Reply:
Added a comment.